blog_posts/6-tips-for-your-next-(remote)-onboarding.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 6 tips for your next (remote) onboarding
+type: list
+tags: career
```

Comment:
I also used `programming` and `jobs` in a similar post about careers, I think we can add them here for ease of searching on the website.